test/src/e2e_vm_tests/test_programs/should_pass/test_contracts/multiple_impl/json_abi_oracle.json

```diff
@@ -20,7 +20,7 @@
       }
     },
     {
-      "logId": 165711,
```

Comment:
Not sure the reason of this change 🤔

Reply:
@mohammadfawaz is working on stabilizing it. Small changes to the code result in all log ids being updated. Not ideal but hopefully temporary.

Reply:
I knew this was gonna cause problems but I didn't know that every other PR is going to be affected 😢